tap_twilio/schemas/cumulative_statistics.json

```diff
@@ -79,91 +79,91 @@
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "min": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "max": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "total": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         }
       }
     },
     "wait_duration_until_accepted": {
       "type": [
         "null",
-        "string"
+        "object"
       ],
       "additionalProperties": false,
       "properties": {
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "min": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "max": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "total": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         }
       }
     },
     "wait_duration_in_queue_until_accepted": {
       "type": [
         "null",
-        "string"
+        "object"
       ],
       "additionalProperties": false,
       "properties": {
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "min": {
           "type": [
             "null",
-            "number"
+            "integer"
```

Comment:
is this OK as integer? Is the wait duration always full measurements? (like seconds?)

Reply:
Yes, they always returned a whole number via their API (in their docs, they do not say anything specific about this)